src/inc/msquichelper.h

```diff
@@ -306,7 +306,9 @@ GetValue(
 {
     const size_t nameLen = strlen(name);
     for (int i = 0; i < argc; i++) {
-        if (_strnicmp(argv[i] + 1, name, nameLen) == 0) {
+        if (_strnicmp(argv[i] + 1, name, nameLen) == 0
+            && strlen(argv[i]) > 1 + nameLen + 1
+            && *(argv[i] + 1 + nameLen) == ':') {
```

Comment:
I think there might be places where we support `-enableFlag` style args too.

Reply:
Ok, it looks like we have a few in the interop code. I'll investigate.